packages/web/src/download/Cover.tsx

```diff
@@ -87,4 +89,7 @@ const styles = StyleSheet.create({
     marginBottom: 20,
     maxWidth: '60vw',
   },
+  zIndex: {
+    zIndex: 2,
```

Comment:
Nit: Consider separating z-index values by some constant (e.g. 100)

Reply:
I changed to constant of 10